Library/Formula/nethogs.rb

```diff
@@ -0,0 +1,32 @@
+class Nethogs < Formula
+  desc "Net top tool grouping bandwidth per process"
+  homepage "https://raboof.github.io/nethogs/"
+  url "https://github.com/raboof/nethogs/archive/v0.8.1.tar.gz"
+  sha256 "4c30ef43814549974a5b01fb1a94eb72ff08628c5a421085b1ce3bfe0524df42"
+
+  if OS.linux?
```

Comment:
The `OS.linux?`/`OS.mac?` modifications are added downstream by Linuxbrew; we don't have the time or resources to support Linux directly.  Please remove anything Linux-specific, thank you!